persist_annotations.py

```diff
@@ -0,0 +1,40 @@
+from sumtool.storage import store_summary_metrics
```

Comment:
This persists the xent annotations by xsum id so that we can look them up easily later